src/mpyl/schema/run_properties.schema.yml

```diff
@@ -1,29 +1,29 @@
 $schema: http://json-schema.org/draft-07/schema#
 $id: schema/run_properties.schema.yml
 required:
-  - build
+- build
```

Comment:
You seem to be doing this change a lot, should it be a formatting or linting rule if possible?

Reply:
Yeah seems like my yaml formatter is different in pycharm. I'll try to disable it and revert these changes